src/trackForMutations.js

```diff
@@ -1,26 +1,36 @@
-export default function trackForMutations(isImmutable, obj) {
-  const trackedProperties = trackProperties(isImmutable, obj);
+export default function trackForMutations(isImmutable, ignore, obj) {
```

Comment:
Yeah, we can do that here. Anyway, not a real concern to me since this is not public API.